webapp/lib/view.dart

```diff
@@ -2,10 +2,16 @@ import 'dart:convert';
 import 'dart:html';
 
 import 'model.dart' as model;
+import 'new_model.dart' as new_model;
+
 
 import 'logger.dart';
 import 'controller.dart' as controller;
 
+import 'platform.dart' as platform;
+
+
+
```

Comment:
accidental leftover edit from development? (I don't see this import being used)

```suggestion
import 'platform.dart' as platform;



```

Reply:
yup - left over from the second phase of FB integration, thanks